### PR TITLE
sammary transaction

### PR DESCRIPTION
**Pull request by thyerre**

### CODE REVIEW

**Comment by thyerre** [comment]
[sammary transaction](https://trello.com/c/9OODL2FL/49-sammary-transaction)

**Review by DevKleber — CHANGES_REQUESTED** [review]
image

File: src/pages/Dashboard/Sammary/index.tsx [hunk 1]
```diff
@@ -70,8 +70,19 @@ export function Sammary({
...
+							<NumberFormat
+								value={lastCard?.value}
+								prefix={'R$ '}
+								displayType={'text'}
+								thousandSeparator="."
+								decimalSeparator=","
+								renderText={value => <Value style={{color: isTransaction ? '#E62E4D' : '#fff'}}>{value}</Value>}
+							/> : 
+							<>
+								{lastCard?.value?.totalPlanejado}%
```
Comment (DevKleber) [inline src/pages/Dashboard/Sammary/index.tsx:83]:
```suggestion
								{lastCard?.value?.totalPlanejado}%
```
```suggestion

```

File: src/pages/Dashboard/Sammary/index.tsx [hunk 2]
```diff
@@ -70,8 +70,19 @@ export function Sammary({
...
+								{lastCard?.value?.total.toFixed(1)}%<De> de </De>
```
Comment (DevKleber) [inline src/pages/Dashboard/Sammary/index.tsx:84]:
```suggestion
						{lastCard?.value?.total.toFixed(1)}%<De> de </De>
						{lastCard?.value?.totalPlanejado}%
```

**Review by DevKleber — CHANGES_REQUESTED** [review]

**Review by DevKleber — APPROVED** [review]

**Pull request merged**